modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -550,4 +553,71 @@ private function fieldFormatSocialElement($items, $body) {
       $body->addChild($social);
     }
   }
+
+  /**
+   * Add ads if configured in settings
+   */
+  public function addAdsFromSettings() {
```

Comment:
Actually never mind, let's save this as a potential follow-up issue, for when we resolve the inheritance vs composition class structure question.